pylint/lint/pylinter.py

```diff
@@ -1276,21 +1275,29 @@ def _report_evaluation(self):
         # syntax error preventing pylint from further processing)
         note = None
         previous_stats = config.load_results(self.file_state.base_name)
-        if self.stats["statement"] == 0:
+        if self.stats.statement == 0:
             return note
 
         # get a global note for the code
         evaluation = self.config.evaluation
         try:
-            note = eval(evaluation, {}, self.stats)  # pylint: disable=eval-used
+            stats_dict = {
```

Comment:
Should this be a function un checker stats ?

Reply:
Don't know. It is only really used here. Might "bloat" the class. Tell me whatever you prefer!

Reply:
Let's keep it that way then.